devbox.go

```diff
@@ -73,3 +73,7 @@ func GlobalDataPath() (string, error) {
 func PrintEnvrcContent(w io.Writer) error {
 	return impl.PrintEnvrcContent(w)
 }
+
+func OnlyPathWithoutWrappers() string {
```

Comment:
What was the reason for skipping adding this to Devbox interface?

Reply:
Its a small utility function which doesn't have any dependency on the core "devbox" struct.

I was going to inline it in `boxcli/shellenv.go`, but needed the `exportify` function so added it here.